packages/bpk-component-graphic-promotion/src/__snapshots__/BpkGraphicPromo-test.tsx.snap

```diff
@@ -108,11 +108,11 @@ exports[`BpkGraphicPromo should render correctly when centre aligned 1`] = `
   >
     <div
       aria-hidden="true"
-      class="bpk-graphic-promo__container bpk-graphic-promo__container--center"
+      class="bpk-graphic-promo__container bpk-graphic-promo__container--start"
```

Comment:
This looks like its changed, but actually its because this snapshot is now output in a different order (reasons unknown) under a new title (line 100) making the diff hard to track.